### ISSUE TITLE
How to set ibus using cutom theme

### ISSUE THREAD

**Issue by tmltmltml** [tuberry/ibus-tweaker#6]
Hello, @tuberry . I'm using ibus-rime input method, and I want to change the color scheme. 
According to the methods I Google, I changed gnome-shell.css(/usr/share/themes/my-theme/gnome-shell/gnome-shell.css), 
  however, after I log out or reboot, it didn't affect the color scheme of ibus.
And when I use ibus-tweaker, it surely can change!
So, could you please tell me ibus-tweaker change which file? 
Thank you in advance!

**Comment by tuberry** [comment]
Cannot reproduce it with Gnome Shell 3.36.5/materia-gtk-theme
```css
/* /usr/share/themes/Materia-compact/gnome-shell/gnomr-shell.css */
.candidate-popup-content {
  padding: 8px;
  spacing: 0;
  color: rgba(12, 120, 56, 0.6);
}
```
![rime](https://user-images.githubusercontent.com/17917040/92352022-b31c4800-f10f-11ea-9593-02f59d9c9852.gif)
1. ibus-tweaker replaces gnome-shell's ibus stylesheet with its [own](https://github.com/tuberry/ibus-tweaker/blob/master/ibus-tweaker%40tuberry.github.com/stylesheet.css), so you should uncheck the theme in ibus-tweaker before applying your custom style;
2. It's not recommanded to change files on system level and better to do that at [$HOME](https://stackoverflow.com/questions/29538867/gnome-shell-theme-css-documentation). Also, welcome to try [user-theme-x](https://github.com/tuberry/user-theme-x)

**Comment by tmltmltml** [comment]
Based on your reply, especially [$HOME](https://linuxhint.com/modify_style_gnome_shell_themes/), I find this problem is absolutely caused by my ignorance.
Specifically, I modified the theme which is applied for `Applications`.  According to [$HOME](https://linuxhint.com/modify_style_gnome_shell_themes/), I modified the theme applied for `Shell` based on your example, then my goal is achieved.
According to your tip 2, I have created a .css file in my $HOME directory.
Last but not least, thanks very much for your reply and your work on ibus -- `ibus-tweaker` and `user-theme-x`!
They saved me a lot time. Thank you!
